### ISSUE TITLE
Broken SplitRecoveryIT

### ISSUE THREAD

**Issue by ctubbsii** [apache/accumulo#1486]
SplitRecoveryIT has been broken in 2.1 builds lately, possibly due to @keith-turner changes in #1462 :

```
12:51:09 [ERROR] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 15.166 s <<< FAILURE! - in org.apache.accumulo.test.functional.SplitRecoveryIT
12:51:09 [ERROR] test(org.apache.accumulo.test.functional.SplitRecoveryIT)  Time elapsed: 14.978 s  <<< FAILURE!
12:51:09 java.lang.AssertionError: expected:<0> but was:<1>
12:51:09 	at org.junit.Assert.fail(Assert.java:88)
12:51:09 	at org.junit.Assert.failNotEquals(Assert.java:834)
12:51:09 	at org.junit.Assert.assertEquals(Assert.java:645)
12:51:09 	at org.junit.Assert.assertEquals(Assert.java:631)
12:51:09 	at org.apache.accumulo.test.functional.SplitRecoveryIT.test(SplitRecoveryIT.java:331)
12:51:09 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
12:51:09 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
12:51:09 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
12:51:09 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
12:51:09 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
12:51:09 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
12:51:09 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
12:51:09 	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
12:51:09 	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
12:51:09 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
12:51:09 	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
12:51:09 	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
12:51:09 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
12:51:09 	at java.base/java.lang.Thread.run(Thread.java:834)
12:51:09 
```

**Comment by ctubbsii** [comment]
1.x might also be affected, since #1462 was also merged into the 1.9 branch.

**Comment by milleruntime** [comment]
This should be fixed when #1501 is merged. I think this was a case where TabletFile was used for part of the code, mismatching types and caused the ImmutableMap to fail.

**Comment by ctubbsii** [comment]
@milleruntime If you're confident that fixed it, I'll go ahead and close this issue. We can reopen if we see it again.
